app/api/custom/orders.py

```diff
@@ -32,36 +32,36 @@
 ticket_blueprint = Blueprint('ticket_blueprint', __name__, url_prefix='/v1/tickets')
 
 
-@ticket_blueprint.route('/<string:order_identifier>')
-@order_blueprint.route('/<string:order_identifier>/tickets-pdf')
+@ticket_blueprint.route('/<int:attendee_id>')
+@order_blueprint.route('/<int:attendee_id>/tickets-pdf')
 @jwt_required
-def ticket_attendee_authorized(order_identifier):
+def ticket_attendee_authorized(attendee_id):
     if current_user:
         try:
-            order = Order.query.filter_by(identifier=order_identifier).first()
+            order = Order.query.filter_by(user_id=attendee_id).first()
```

Comment:
Not really. As I already told you, Order.user_id != Attendee.id

Attendee == TicketHolder

Attendee != Order.user

You need to understand what is Ticket Buyer and what is an attendee or a ticket holder. Then understand why the issue was raised and what is the actual problem. We discussed this in meeting as well. And then think if the changes actually solve the problem

Reply:
this is my fault, I missed to change this. Will update the PR

Reply:
Ticket Buyer is the person who is ordering the ticket
Ticket Holder = Can be ticket buyer or some other person

Issue was raised because on frontend download tickets, a random ticket was downloaded - to solve this we will have to save ticket attendee wise.

Reply:
I don't understand why you removed the code. Why was it necessary. How is it compatible with previous behaviour?

Reply:
I removed the code as the ticket pdf is now created and saved attendee wise so the ticket path needs to be changed for every ticket holder. The same common code is now moved to create pdf function.

Reply:
It still wasn't constant. It changed with order. Now it changes with ticket holder. I understand that. But you removed the code of sending the attachments and other things. We need that. It should be adapted to recent changes

Reply:
I have a question, shall the invoice path and ticket path then be sent as attatchement to the function as earlier? or should I add a parameter attachment in function and append that function parameter value to the invoice, ticket_path, ?

Reply:
Also, I think that send_Email function should not have this code as it's a generic function? What are your views

```  for holder in order.ticket_holders:```